src/resolve-script-object-to-string.js

```diff
@@ -12,13 +12,22 @@ function resolveScriptObjectToString(script) {
 
 function resolveScriptObjectToScript(script) {
   if (isPlainObject(script)) {
-    if (!isUndefined(script.script)) {
-      return script
-    } else if (!isUndefined(script.default)) {
-      return resolveScriptObjectToScript(script.default)
-    }
+    return resolvePlainObjectToScript(script)
   } else if (isString(script)) {
     return {script}
   }
   return undefined
 }
+
+function resolvePlainObjectToScript(script) {
+  if (!isUndefined(script.hiddenFromHelp)) {
```

Comment:
I think this should probably simply be: `if (script.hiddenFromHelp) {`

I think we're good to go with `truthy`. So `hiddenFromHelp: false` should not hide it from help ;-)